web/src/runtime.ts

```diff
@@ -2432,3 +2444,28 @@ export async function hasNDArrayInCache(
   list = list["records"] as Array<NDArrayShardEntry>;
   return await artifactCache.hasAllKeys(list.map(key => new URL(key.dataPath, ndarrayCacheUrl).href));
 }
+
+/**
+ * Given cacheUrl, search up items to delete based on cacheUrl/ndarray-cache.json
+ *
+ * @param ndarrayCacheUrl
+ * @param cacheScope
+ */
+export async function deleteNDArrayCache(
+  ndarrayCacheUrl: string,
```

Comment:
We should replace `ndarrayCacheUrl` to `cacheUrl` to match the docstring and the semantics.